documentation/index.json

```diff
@@ -60,14 +60,14 @@
                     "subpath": "processors.adoc"
                 },
                 {
-                    "title": "PIP",
-                    "description": "The Product Information Portal (PIP) for Raspberry Pi compliance documents",
+                    "title": "Product Information Portal",
+                    "description": "Raspberry Pi compliance documents",
```

Comment:
Isn't the intention for PIP to (eventually) be more than just compliance documents?

Reply:
Yes. Eventually, most of what's on datasheets.raspberrypi.com will but put into PIP. But not before it's overhauled to get human-readable URLs and allow us to use human-readable document names. A string of numbers and letters which you have to go and look up isn't helpful. Documents should have human-readable names.

Reply:
The names are automatically generated by Cognidox. Not sure there is a way round that.

Reply:
Sure, but PIP isn't raw Cognidox. Paul and Chris are going to look at it.